docs/content/docs/bundlers.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Fluid and Bundlers
+aliases:
+  - "/start/bundlers/"
+---
+
+The following are known bundler issues, answers to some of the most commonly asked questions about the Fluid
+Framework and Bundlers.
+
+## Known Errors with Webpack 5
+We know the following packages cause errors; assert, buffer, events and url.
+
+This issue causes compiling any project that uses these libraries to fail. Upgrading to Webpack 5 removed the automatic NodeJS polyfills. We now have to manually install the missing packages and add the fallback.
+
+To fix this, run the command `npm install -D <package>` and add the fallback `resolve.fallback: { "<package>": require.resolve("<package>/") }` to the affected package's webpack.config.js. An example of this is located [here](https://github.com/microsoft/FluidFramework/blob/a4c38234a920abe9b54b1c26a14c0a8e430cd3fa/packages/tools/webpack-fluid-loader/webpack.config.js#L37)
+
+Error:
+```bash
+BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
```

Comment:
Again here, is this an internal doc or one aimed at external audiences? If it is external, why call this a breaking change? Why assume they already had an FF project with WP 4?

Reply:
Is this example error output we get from Webpack? If so, we should clarify that a bit more. We definitely want to be clear that we are surfacing common issues from *Webpack's* breaking change, and not noting a breaking change of our own.

Reply:
Yes, this is what webpack spits out when you have dependencies on node core modules. I think we need to have the error _somewhere_ in the docs to help people find it.

Reply:
Is there anything else to do here or should I mark this as resolved?

Reply:
I think we at least need to clarify the label we're using before the sample error. Right now it just says "Error:", but I think we need give a bit more context into when the error is expected to occur.

Reply:
Change looks good to me. @Rick-Kirkham do you have any remaining concerns here?